setup.py

```diff
@@ -22,7 +22,7 @@
                     'imageio',
                     'appdirs',
                     'scooby>=0.5.1',
-                    'meshio>=3.3.0',
+                    'meshio>=4.0.3',
```

Comment:
`meshio` is on 4.0.7 as of this morning... it seems like lots of patches are coming out lately. 

What's the most stable version that we should set as the minimum here? Thoughts, @nschloe? I want to make sure we set it as the "most stable" one since it looks like there may have been some issues with a few of the latest releases.

Reply:
> What's the most stable version that we should set as the minimum here?

There is no such thing as "most stable"? If someone reports a bug fix, I'm never shy to release the fix straight away. If you have issues with 4.0.1, we'll try to find the bug and fix it, and you should set 4.0.2 as a minimum. Until that happens, I'd simply go with `>=4.0.0, < 5.0.0`.

Reply:
``4.0.3`` fixes the ``ModuleNotFoundError`` when running Python 3.5 tests, so that should be the minimum requirement to pass the tests for ``pyvista`` ;)

``meshio`` increments its version following [semver](https://semver.org/) so ``4.0.7`` is as *stable* as ``4.0.3``.

Reply:
@keurfonluu, should we add in `< 5.0.0` too?

Reply:
Better safe than sorry :)